test/extended/networking/commatrix.go

```diff
@@ -0,0 +1,163 @@
+package networking
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift-kni/commatrix/commatrix"
+	"github.com/openshift-kni/commatrix/consts"
+	"github.com/openshift-kni/commatrix/debug"
+
+	clientutil "github.com/openshift-kni/commatrix/client"
+	nodesutil "github.com/openshift-kni/commatrix/nodes"
+	"github.com/openshift-kni/commatrix/types"
+
+	configv1 "github.com/openshift/api/config/v1"
+	configv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = g.Describe("[sig-network][Feature:commatrix][Serial]", func() {
+	g.It("should apply firewall by open ports on nodes", func() {
```

Comment:
I think better description is needed here, it's blocking all ports except the ones OCP is listening on